framework/src/interfacekernels/InterfaceReaction.C

```diff
@@ -34,13 +34,13 @@ InterfaceReaction::computeQpResidual(Moose::DGResidualType type)
   switch (type)
   {
     // Move all the terms to the LHS to get residual, for primary domain
-    // Residual = kf*u - kb*v = kf*u - kb*v
-    // Weak form for primary domain is: (test, kf*u - kb*v)
+    // Residual = -kf*u + kb*v
+    // Weak form for primary domain is: (test, -kf*u + kb*v)
     case Moose::Element:
       r = _test[_i][_qp] * (_kf * _u[_qp] - _kb * _neighbor_value[_qp]);
```

Comment:
This change doesn't make sense. The residual is clearly `ku*f - kb*v`

The comment is a little confusing in that it says the "weak form is `(test, kf*u - kb*v)`" since parentheses are typically reserved for integrations over volumes while angle brackets are used for integrations over surfaces

Reply:
umm I ll have to re-think this

Reply:
ditched it

It was just a confusion in how I read the docs. 
dC/dt = -Reaction rate as the reaction rate is a loss term